lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -5,7 +5,7 @@
  */
 'use strict';
 
-/* global DOM, ViewerUIFeatures, ReportRenderer, DragAndDrop, GithubApi, PSIApi, logger, idbKeyval */
+/* global DOM, ViewerUIFeatures, ReportRenderer, DragAndDrop, GithubApi, PSIApi, logger, idbKeyval, __LIGHTHOUSE_JSON__ */
```

Comment:
we don't use it as a global, I think you just need to do some typescript fanagling